src/main/java/tests/testCase.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+import loginPage.Homepage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class testCase {
+    WebDriver driver = new FirefoxDriver();
+    Homepage page;
+    @BeforeMethod
+    public void browserSetup() {
+        page = new Homepage(driver);
+        driver.get("https://www.saucedemo.com/");
+    }
+    @Test
+    public void runningElements() {
```

Comment:
this method doesn't say what the test is actually doing. It would be better named something like "logInToMyAccountTest"